ee/clickhouse/sql/person.py

```diff
@@ -299,21 +299,11 @@
 ORDER BY count DESC, key ASC
 """
 
-GET_PERSONS_FROM_EVENT_QUERY = """
-SELECT
-    person_id,
-    created_at,
-    team_id,
-    person_props,
-    is_identified,
-    arrayReduce('groupUniqArray', groupArray(distinct_id)) AS distinct_ids
+GET_ACTORS_FROM_EVENT_QUERY = """
+SELECT 
+    {id_field} AS id
```

Comment:
Nit: let's name this `AS actor_id` - this way we can be consistent in other queries. id is IMO overloaded.